src/components/admin/BackgroundSelect.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { UseFormRegister } from "react-hook-form";
+import { useAsyncFn } from "react-use";
+import { ImageInput } from "components/admin/ImageInput";
+import { ImageInputChangeProps } from "components/admin/ImageInput/ImageInput";
+import { InputGroup } from "components/admin/InputGroup";
+import { noop } from "lodash/fp";
+
+import { DEFAULT_BACKGROUNDS } from "settings";
+
+import { updateMapBackground } from "api/admin";
+
+import { WorldId } from "types/id";
+import { AnyForm } from "types/utility";
+
+import { useUserId } from "hooks/user/useUserId";
+
+import { Loading } from "components/molecules/Loading";
+import { SubmitError } from "components/molecules/SubmitError";
+
+interface BackgroundSelectProps {
+  venueId: string;
+  venueName: string;
+  spaceSlug: string;
+  worldId: string;
```

Comment:
[Q] Can some of these strings here and elsewhere be replaced with branded ones (e.g. `SpaceId`, `WorldSlug`)? 

If too much work for this PR, maybe a follow up one.